### DISCUSSION TITLE
Import trace for requested module

### DISCUSSION THREAD

**Discussion by AnonSols** [vercel/next.js#61968] [Help] [answered]
### Summary

I'm trying npm run dev on a fresh install I can't get the site running locally at all. Just getting piles of errors, ending in

Import trace for requested module:
./node_modules/next/dist/client/components/async-local-storage.js
./node_modules/next/dist/client/components/request-async-storage.external.js
./node_modules/next/dist/client/components/redirect.js
./node_modules/next/dist/client/components/redirect-boundary.js
./node_modules/next/dist/client/components/app-router.js

.
.
.

On load time it shows the next landing page, after load time is completed, then I get an application error.

I've tried countless solutions on a 'Hydration error', which I get from the console of such, doesn't seem to work though.
I've tried updating my browser.
I've installed the encoding from npm.
I've downgraded my version of node to 18.15.0
I've uninstalled node and vscode, honestly just had to try that as well.
I've turned off browser extension, opened the url in different browser and even tried with incognito mode as well. Nothing WORKS!!!!

Please, ANYONE how can this ever be fixed??!!

### Additional information

```js
This happens to any new fresh installation of nextjs app
```

**Comment by Eazyisreal** [comment]
I have the exact same problem and I was hoping to find a solution on here. Any luck on fixing the error ?

**Reply by AnonSols** [reply, marked as answer]
RUN YOUR APPLICATION USING THE WINDOWS COMMAND PROMPT AND NOT POWERSHELL!(If running on windows)
I currently don't know why it works that way, but it supprisengly worked perfectly. 
Before then I was using bash and powershell.

**Reply by jordansmalls** [reply]
This works...

**Reply by Ronexan** [reply]
Wow! This actually works!

**Reply by Nivnivu** [reply]
its so wierd but its working, thank you after 50 minutes of searching ..
